lib/popup.html

```diff
@@ -5,22 +5,39 @@
      <link rel="stylesheet" href="../assets/popup.css">
   </head>
   <body>
-    <header>
-      <img src="../assets/white_logo.png" alt="Qowala logo"/>
-      <span>Hey, it's your friends at Qowala!</span>
-      <div></div>
-    </header>
-    <div class="non-measure-display">
-      <h3>This website is not tracked!</h3>
-    </div>
-    <div class="measure-display">
-      <h3>You've spent...</h3>
-      <div class="time-spent"></div>
-      <div class="parameters">
-        On <select class="parameters__dropdown network"></select>
-        <select class="parameters__dropdown date"></select>
+    <main id="app">
+      <header>
+        <img src="../assets/white_logo.png" alt="Qowala">
+        <hr>
+      </header>
+      <div v-if="tracked">
+        <p class="text secondary">
+          You've spent…
+        </p>
+        <div class="time-spent">
+          {{ timeSpent }}
+        </div>
+        <div class="parameters">
+          On
+          <select v-model="hostname" class="parameters__dropdown network">
+            <option v-if="noData" disabled :value="hostname">Please select another site</option>
```

Comment:
If it is maybe better to remove this and the `noData` variable. It seems the only moment you see this, is when you change from **"Today"** to **"Yesterday"** which displays **"Please select another site"** as default value in the dropdown.
I don't see any other cases where it can be useful. So if you agree, we can remove this.